### DISCUSSION TITLE
Guys, para aqueles que não querem perder 5 horas configurando Next.js + Jest + Story Book + Sass, aqui um Boilerplate :)

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#2450] [Mostre seu trabalho]
Opa, galera!

Sei como pode ser desafiador e demorado configurar um novo projeto em Next.js, especialmente quando você quer integrar ferramentas como Jest, Storybook, Sass, e mais... 

Pensando nisso, estou super empolgado em compartilhar com vocês o Next.js Pro Boilerplate! (bom, precisava de um nome rsrs)

Link para o repo:
https://github.com/wilsonneto-dev/Next.js-Pro-Boilerplate

Este boilerplate é aideal para quem quer economizar tempo e evitar a dor de cabeça das configurações iniciais. Com ele, você pode iniciar seu projeto em minutos, não horas! 🕒

### O que está incluído? 📦

- Eslint & Prettier: Para garantir um código limpo e consistente.
- TypeScript: Aproveite o poder da tipagem em seus projetos.
- NextJS: O framework que você já conhece e adora, pronto para produção.
- Sass: Escreva CSS com mais poder e elegância.
- Storybook: Crie componentes de UI isolados para uma melhor experiência de desenvolvimento.
- Jest & React Testing Library: Para testes robustos e confiáveis.

### Como começar? 🚀

É simples! Siga estes passos:

```bash
npm install
npm run dev
```

E voilà! Você está pronto para começar a desenvolver.

Espero que este boilerplate ajude você tanto quanto me ajuda no dia a dia. 

Estou ansioso para ver o que vocês vão criar com ele! Para qualquer dúvida ou feedback, fiquem à vontade para responder a este post.

Good ahcking! 💻🚀

**Comment by ghost** [comment]
Muito bom, estava buscando isso mesmo, obrigada!

**Comment by felquis** [comment]
Só pra vc saber tentei rodar aqui só pra ver qual é que é, e da este erro:
https://nextjs.org/docs/messages/failed-loading-swc 

tentei com node 18 e 20

```
npm run dev

> w25-site@0.1.0 dev
> next dev

   ▲ Next.js 14.0.3
   - Local:        http://localhost:3000

 ⚠ Attempted to load @next/swc-darwin-arm64, but an error occurred: dlopen(/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node, 0x0001): tried: '/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node' (code signature invalid in <FF579EB6-B527-3586-82CE-52BB11CC1B1C> '/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node' (errno=85) sliceOffset=0x00000000, codeBlobOffset=0x067CCED0, codeBlobSize=0x000CFA28), '/System/Volumes/Preboot/Cryptexes/OS/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node' (no such file), '/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node' (code signature invalid in <FF579EB6-B527-3586-82CE-52BB11CC1B1C> '/Users/user/projects/Next.js-Pro-Boilerplate/node_modules/@next/swc-darwin-arm64/next-swc.darwin-arm64.node' (errno=85) sliceOffset=0x00000000, codeBlobOffset=0x067CCED0, codeBlobSize=0x000CFA28)
 ⨯ Failed to load SWC binary for darwin/arm64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
```
